spec/models/manageiq/providers/google/cloud_manager/provision_workflow_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe ManageIQ::Providers::Google::CloudManager::ProvisionWorkflow do
+  include Spec::Support::WorkflowHelper
+
+  let(:admin) { FactoryGirl.create(:user_with_group) }
+  let(:ems) { FactoryGirl.create(:ems_google) }
+  let(:template) { FactoryGirl.create(:template_google, :name => "template", :ext_management_system => ems) }
+  let(:workflow) do
+    stub_dialog
+    allow(User).to receive_messages(:server_timezone => "UTC")
+    wf = described_class.new({:src_vm_id => template.id}, admin.userid)
+    wf
```

Comment:
No need to set the `wf` variable only to return it on the next line.  Just return the result from `described_class.new`.